.env

```diff
@@ -0,0 +1,3 @@
+API_KEY=tBUTDQ6rF8MkOy8X
+SPACE_ID_1=gL2sTLbVCqghDwB8/gathertown_js
+SPACE_ID_2=tiWyyxlRs6Bl1dVS/test-space-1

```

Comment:
This is supposed to be not included on your PR @princejoogie can you please remove this and also add a .env file to our .gitignore file.

Reply:
oops, disnt notice this got added into the commit. this is noted